### DISCUSSION TITLE
ISRC & UPC: Chart sales counts

### DISCUSSION THREAD

**Discussion by simonv3** [funmusicplace/mirlo#372] [Ideas]
[discussion](https://discord.com/channels/1070731899317796974/1100766224553627759/threads/1185519100643000380)
[How to get an ISRC & UPC code](https://releese.io/article/how-to-get-an-isrc-and-upc-code-for-a-music-release/)

This is important for any artists who would be aiming to get on any charts. I wonder if meta information already contains this for some songs, and whether we can extract it. 

Bandcamp just lets artists enter the code, and presumably they then report the sales too as a licensed retailer.

**Comment by paper-alpaca** [comment]
I put these information in my tracks, so yes, you will find it in the metadata
